frontendGraphApi/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/labeling/SyncToRemoteStatusLabelGenerator.java

```diff
@@ -18,7 +19,8 @@ public static String getLabel(SyncToRemoteStatus.Relation relation, String remot
         Case($(Untracked), "untracked"),
         Case($(Ahead), "ahead of " + remoteName),
         Case($(Behind), "behind " + remoteName),
-        Case($(Diverged), "diverged from " + remoteName),
+        Case($(DivergedAndNewerThanRemote), "diverged from " + remoteName),
```

Comment:
To avoid clutter: `diverged from & newer than` is kinda the default (unless you only ever do reviews and no coding yourself xD) and `diverged from & older than` is typically more of an exceptional situation

Reply:
I tried `diverged from & newer than` in CLI at first but it was overly verbose

Reply:
Fine!  I think that a short comment about it would be nice :speech_balloon: